tests/e2e/standalone/commands.go

```diff
@@ -119,12 +120,19 @@ func cmdStop(appId string, args ...string) (string, error) {
 }
 
 // cmdUninstall uninstalls Dapr with --all flag and returns the command output and error.
-func cmdUninstall() (string, error) {
+func cmdUninstall(args ...string) (string, error) {
+	uninstallArgs := []string{"uninstall", "--all"}
```

Comment:
Can we pre-allocate the size for uninstallArgs? I think we should be able to and if un-used due to if-else branches, we can keep a counter `n` and return uninstallArgs[:n]. As far as I understand, append further will not have O(1) time complexity: https://medium.com/vendasta/golang-the-time-complexity-of-append-2177dcfb6bad

Reply:
I agree that this should be done.... 
But I belive this can be done later in master.
This is specifically test code...

Reply:
Since the performant benefit won't be much for this scenario, I would lean towards readability/maintainability over making tiny performance improvement in this e2e test. Please let me know your thoughts!

Reply:
Though I don't agree with the readability part, but I agree with being in e2e test and number of params being small, so will go ahead and approve and merge